module/bdev/aio/bdev_aio.c

```diff
@@ -234,18 +321,138 @@ bdev_aio_writev(struct file_disk *fdisk, struct spdk_io_channel *ch,
 	return len;
 }
 
+static struct aio_request_ctx *
+create_aio_request_ctx(struct spdk_bdev_io *bdev_io, aio_request_fn fn, size_t ctx_size)
+{
+	struct aio_request_ctx *request = malloc(sizeof(struct aio_request_ctx) + ctx_size);
```

Comment:
As mentioned by @gila in https://github.com/openebs/spdk/pull/10#r591365095, it's convention in SPDK to use `calloc` to avoid UB.

Reply:
This has been done.